frontend/components/buttons/Button/_styles.scss

```
@@ -217,71 +217,37 @@ $base-class: "button";
     }
   }
 
-  &--text-link {
-    @include button-variant(transparent);
+  // Looks exactly like a CustomLink but is a <button>
+  &--link {
+    @include link;
+    @include animated-bottom-border($core-fleet-black, $ui-fleet-black-25);
+    background: transparent;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)